### ISSUE TITLE
Dev Recruitment

### ISSUE THREAD

**Issue by not-a-dev0** [peatio/peatio#845]
We are looking to recruit some devs for production deployment of custom peatio solution. We are a startup and can't pay much at the moment but we are willing to negotiate favorable terms that will allow you to own part of the company/exchange. 

please email: ilfiore2017@gmail.com

**Comment by Thedabest** [comment]
do you want frontend dev?

**Comment by not-a-dev0** [comment]
@Thedabest Yes, We are hiring frontend and backend developers. Email me ilfiore2017@gmail.com

**Comment by magicTurtle-7** [comment]
I am a full stack developer with a lot of experience in PHP/OOP, HTML, JavaScript, Json and other languages
Here is a video so you can see that I have installed Peatio earlier https://www.youtube.com/watch?v=SOKwkIJlLDs
